modules/ROOT/pages/custom-policy-4-reference.adoc

```diff
@@ -195,15 +194,15 @@ One of the main additions is that now policies can also be applied on outbound H
 ----
 
 <1> Same `http-policy:proxy` element is used for both types of policies. Both source and operation blocks may be present. At least one of them is mandatory. When both are present things like sharing variables between them are possible to keep state.
-<2> Operation is where the logic that is going to be executed around the flow’s HTTP Requester is placed. If flow does not contain that element, then this block is never executed. It allows to inject behaviour before an HTTP Request is sent to a proxy’s backend and after an HTTP Response is received by the proxy. Just like source block, the execute-next element divides when processors are executed.
+<2> Operation is where the logic that is going to be executed around the flow’s HTTP Requester is placed. If flow does not contain that element, then this block is never executed. It is allowed to inject behaviour before an HTTP Request is sent to a proxy’s backend and after an HTTP Response is received by the proxy. Just like source block, the execute-next element divides when processors are executed.
```

Comment:
Similar to the instance above, you are changing the meaning of the sentence:
> It is allowed to inject behaviour before an HTTP Request is sent to a proxy’s backend and after an HTTP Response is received by the proxy. Just like source block, the execute-next element divides when processors are executed.

Should be:

> This element allows you to inject a behavior before an HTTP Request is sent to a proxy’s backend and after an HTTP Response is received by the proxy. Just like source block, the execute-next element divides when processors are executed.